Obsidian.API/_Attributes/RequirePermissionAttribute.cs

```diff
@@ -19,6 +19,11 @@ public RequirePermissionAttribute(PermissionCheckType checkType = PermissionChec
 
         public override Task<bool> RunChecksAsync(CommandContext context)
         {
+            if (context.Player == null)
+            {
+                throw new Exception("Player not found");
```

Comment:
Not sure if this makes sense here since the possibility that the `Player` is `null` can always happen and might be expected when the console executes a command which uses this check. Also meaning that this code is put in the wrong place, instead make it return false like the code below and put it under the [console issuer check](https://github.com/ObsidianMC/Obsidian/pull/168/files#diff-8cda4a03e94a2cc8a5b3572e4351d6821f7b78359d098a1075bff1be6caf626eR27-R28).

Reply:
I have amended this, good spot, and thank you for the context!